### PR TITLE
Eliminate redundant Win32 suffix in Windows embedder

### PR DESCRIPTION
**Pull request by cbracken**
Eliminate Win32 suffixes in Windows embedder

Now that we've removed the UWP embedder, eliminate remaining Win32
suffixes on identifiers and _win32 suffixes from filenames.

This renames all files and types ending in _win32/Win32 that don't
require further rework/merging/simplification.

No additional tests since this is a renaming only with no semantic change.

Issue: https://github.com/flutter/flutter/issues/108386

## Pre-launch Checklist

- [X] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [X] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on
writing and running engine tests.
- [X] I updated/added relevant documentation (doc comments with `///`).
- [X] I signed the [CLA].
- [X] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/main/CONTRIBUTING.md#style
[testing the engine]: https://github.com/flutter/flutter/wiki/Testing-the-engine
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: shell/platform/windows/flutter_windows_engine.cc [hunk 1]
```diff
@@ -523,8 +522,6 @@ bool FlutterWindowsEngine::DispatchSemanticsAction(
 }
 
 void FlutterWindowsEngine::UpdateSemanticsEnabled(bool enabled) {
-  using AccessibilityBridgeDelegateWindows = AccessibilityBridgeDelegateWin32;
```
Comment (cbracken) [inline shell/platform/windows/flutter_windows_engine.cc:526]: I should probably have cleaned this up when I deleted the #ifdef'ed Win32/UWP; no longer necessary either way.

**Comment by cbracken** [comment]
The licence script will fail; I'll push an update for that once that happens :)

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

**Review by yaakovschectman — APPROVED** [review]

File: shell/platform/windows/flutter_window.h [hunk 2]
```diff
@@ -23,12 +23,12 @@ namespace flutter {
 // the future, there will likely be a CoreWindow-based FlutterWindow as well.
```
Comment (loic-sharma) [inline shell/platform/windows/flutter_window.h:23]: `CoreWindow` is UWP, right? This comment seems stale, can we delete it?

File: shell/platform/windows/settings_plugin_win32.h [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 #include <Windows.h>
 
-#include "flutter/shell/platform/windows/event_watcher_win32.h"
+#include "flutter/shell/platform/windows/event_watcher.h"
 
 namespace flutter {
```
Comment (loic-sharma) [inline shell/platform/windows/settings_plugin_win32.h:15]: Should this type & file be renamed as well?

**Comment by loic-sharma** [comment]
For my own knowledge (feel free to punt this for later!), should these files be updated as well? Or are their win32 suffixes not redundant for some reason?

* keyboard_manager_win32.cc
* keyboard_manager_win32.h
* keyboard_win32_common.cc
* keyboard_win32_common.h
* keyboard_win32_common_unittests.cc
* keyboard_win32_unittests.cc
* platform_handler_win32.cc
* platform_handler_win32.h
* platform_handler_win32_unittests.cc
* task_runner_win32.cc
* task_runner_win32.h
* task_runner_win32_window.cc
* task_runner_win32_window.h

**Review by loic-sharma — APPROVED** [review]

**Comment by cbracken** [comment]
> Or are their win32 suffixes not redundant for some reason?

Those need to be done, but will be done in separate one-by-one PRs since they don't fall into the "files and types ending in _win32/Win32 that don't require further rework/merging/simplification" in this PR. Sorry for not calling that out more explicitly in the PR description. I'll edit comment to clarify in the merge.

**Comment by loic-sharma** [comment]
Gotcha that makes sense! :)

**Pull request merged**